sled-agent/config-reconciler/src/dataset_serialization_task.rs

```diff
@@ -180,7 +180,7 @@ pub(crate) struct DatasetTaskHandle(mpsc::Sender<DatasetTaskRequest>);
 impl DatasetTaskHandle {
     // For testing, create a handle on which requests will always fail with a
     // `DatasetTaskError`.
-    #[cfg(any(test, feature = "testing"))]
```

Comment:
Is this "testing" feature still being used?

Reply:
It is in other places, but this method isn't accessible outside the crate, so clippy complained about it being dead code.